server/datastore/mysql/migrations/tables/20230516152807_AddRefetchCriticalQueriesUntilToHosts.go

```diff
@@ -0,0 +1,22 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230516152807, Down_20230516152807)
+}
+
+func Up_20230516152807(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+ALTER TABLE hosts ADD COLUMN refetch_critical_queries_until TIMESTAMP NULL;
```

Comment:
I know we want to avoid adding stuff to the `hosts` table, but it is mainly due to writes to that table that cause locking issues. In this case, for this field, the writes are very infrequent - when triggering the webhook for the MDM migration the timestamp is set, and when clearing it, it is not even a new write, it is part of the "detail queries updated" update on the host.